CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ For information on changes in released versions of Teku, see the [releases page]
  - Reduced memory usage and GC pressure created while tracking the latest attestations for each validator.
  - Reduced CPU and GC pressure during epoch processing by avoiding setting validator effective balances to an unchanged value.
  - Reduced memory usage and GC pressure created by state caches.
+ - Removed deprecated constants from source code. 
```

Comment:
Since this change doesn't have any effect on end users, we don't need to note it in the changelog.